### PR TITLE
feat(reload): add reload component and styles

### PR DESCRIPTION
**Pull request by ami**
⌒°(ᴖ◡ᴖ)°⌒

fixes https://github.com/robinjoseph08/pokedextracker.com/issues/158

### CODE REVIEW

File: app/styles/main.scss [hunk 1]
```diff
@@ -323,6 +323,18 @@ form {
   }
 }
 
+reload div {
+  text-align: center;
+  padding: 10px 30px;
+  background-color: lighten($brand-primary-light, 15%);
```
Comment (ami) [inline app/styles/main.scss]: i can't decide between this color and `$gray-light`. can you try both and let me know what you prefer?

Comment (robinjoseph08) [inline app/styles/main.scss]: im assuming its just `$gray-light` and not `lighten($gray-light, 15%)` cause the latter is pretty much white lol. but i def prefer the yellow for sure 👍

Comment (ami) [inline app/styles/main.scss]: lolz yes that's what i meant. great!

**Comment by robinjoseph08** [comment]
@ami added the functionality! its a tad hard to test locally though (since we dont actually want it to be annoying and stuff when developing). i can either tell you the steps to test it locally ooooooor you can look over the code and then we deploy to staging and we can test it there? whatever works for you

**Comment by ami** [comment]
eh. i'm sure it's fine! MERGGGGGGE

**Pull request merged**